samples/drivers/watchdog/prj.conf

```diff
@@ -1 +1,2 @@
 CONFIG_WATCHDOG=y
+CONFIG_WDT_NRFX=y
```

Comment:
It does not seem to be a good idea to make this sample "Nordic only".

Reply:
It would require to enable Nordic's watchdog in the other way

Reply:
I am not sure I understand. Is it intentional to make this sample to build only for Nordic chips?
The `WDT_NRFX` will not be available for other chips, since it depends on `SOC_FAMILY_NRF`, so cmake will complain.

Reply:
Alternatively, you can add `default y` to the `WDT_NRFX` definition. Then this option will be automatically activated when the watchdog is enabled and the build is done for a Nordic SoC.

Reply:
Yes, that's it. This definition has to be set to ` y ` somewhere. Your idea with setting ` default y ` looks good. I'll update test's ` proj.conf ` file too.